content/python/encyclopedia/replace-in-python.md

```diff
@@ -0,0 +1,90 @@
+<!--
+---
+Title: "Replace in Python" 
+Subjects:
```

Comment:
As of earlier this afternoon, we're making Subjects and tags all lowercase. There's no way you could have known that that's the formatting we would want until now. My bad!

Reply:
no problem, thanks for the heads up! Changed to lowecase